README.md

```diff
@@ -37,7 +39,7 @@ steps:
   - label: "Pipeline upload"
     command: "buildkite-agent pipeline upload"
     plugins:
-      - sparse-checkout:
+      - sparse-checkout#v1.0.0:
           no_cone: true
           paths:
             - .buildkite
```

Comment:
Can probably just remove this example. One is enough

Reply:
Done